src/presenter/films-list-presenter.js

```diff
@@ -25,24 +33,50 @@ export default class FilmsListPresenter {
 
   prepearFilms = (films) => films;
 
+
+  #renderedFilmCount = TASK_COUNT_PER_STEP;
+
+  #loadMoreButtonComponent = new LoadMoreButtonView();
+
   init = (filmsModel, commentsModel) => {
-    this.filmsModel = filmsModel;
-    this.commentsModel = commentsModel;
-    this.cards = this.prepearFilms([...this.filmsModel.getFilms()]);
+    this.#filmsModel = filmsModel;
+    this.#commentsModel = commentsModel;
+    this.#cards = this.prepearFilms([...this.#filmsModel.get()]);
+
+    render(this.#filmsListView, this.#container.element);
 
-    render(this.filmsListView, this.container.getElement());
     render(
       this.filmsListContainerView,
-      this.filmsListView.getElement()
+      this.#filmsListView.element
     );
 
-    this.cards.forEach((film) => {
-      this.createFilm(film);
+    this.#cards.slice(0, 5).forEach((film) => {
+      this.#createFilm(film);
     });
+
+    render(this.#loadMoreButtonComponent, this.#filmsListView.element);
```

Comment:
Рендерить кнопку "показать ещё" имеет смысл только если количество больше чем 5
```suggestion
    if (this.#cards.length > this.#renderedFilmCount) {
      render(this.#loadMoreButtonComponent, this.#filmsListView.element);
      this.#loadMoreButtonComponent.element.addEventListener('click', this.#handleLoadMoreButtonClick);
    }
```

Reply:
Это и решит проблему с отображением показать ещё в 2 листах снизу